cmd/api/README.md

```diff
@@ -13,6 +13,11 @@
 | DB_ENABLE_AUTO_MIGRATION | Auto-migrate the database on startup (create/update schemas). For further details, refer to <https://gorm.io/docs/migration.html> | true (default)                               |
 | PROFILING                | Enable profiling server                                                                                                           | false  (default)                             |
 | PROFILING_PORT           | Profiling Server Port                                                                                                             | 6060  (default)                              |
+| DB_MAX_IDLE_CONNECTIONS  | The number of idle database connections to keep open                                                                              | 10 (default)                                 |
+| DB_MAX_OPEN_CONNECTIONS  | The maximum number of database connections, for best performance it should equal DB_MAX_IDLE_CONNECTIONS                          | 10 (default)                                 |
```

Comment:
This is now the Postgres driver default, which is 100.

Reply:
well according to golang's database/sql package it is 2:
```
const defaultMaxIdleConns = 2

func (db *DB) maxIdleConnsLocked() int {
	n := db.maxIdleCount
	switch {
	case n == 0:
		// TODO(bradfitz): ask driver, if supported, for its default preference
		return defaultMaxIdleConns
	case n < 0:
		return 0
	default:
		return n
	}
}
```

and max open is unlimited (which can also have ramifications, vs. having a fixed number) 

```
// SetMaxOpenConns sets the maximum number of open connections to the database.
//
// If MaxIdleConns is greater than 0 and the new MaxOpenConns is less than
// MaxIdleConns, then MaxIdleConns will be reduced to match the new
// MaxOpenConns limit.
//
// If n <= 0, then there is no limit on the number of open connections.
// The default is 0 (unlimited).
func (db *DB) SetMaxOpenConns(n int) {
	db.mu.Lock()
	db.maxOpen = n
	if n < 0 {
		db.maxOpen = 0
	}
	syncMaxIdle := db.maxOpen > 0 && db.maxIdleConnsLocked() > db.maxOpen
	db.mu.Unlock()
	if syncMaxIdle {
		db.SetMaxIdleConns(n)
	}
}
```

I'm not finding any postgresql specific overrides in the vendor tree to those @sayan-biswas .... for my education can you tell me where you see that 100.

Regardless, it certainly no longer is 10, and I need to update.  Thanks.

Reply:
@sayan-biswas - per the WG meeting today with myself, @avinal and @khrm we at least agreed that the `database/sql` defaults I noted above are the ones that we should document, as there are many flavors of postgresql driver, and those different drivers will have different defaults.

I will try to elaborate on this as part of the README update I am working on.  Unless unexpectedly interrupted I should have some form of this up in an hour or so I think.